### DISCUSSION TITLE
Help with startup

### DISCUSSION THREAD

**Discussion by mark-jones-rga** [MatthewJohn/terrareg#72] [Q&A] [answered]
Hey all, we were running an older "forked" version of terrareg.  We synced a lot of it up and it is working, but when I did a complete sync of the repo with the updated Terrareg, I am receiving an error on AWS ecs task startup and then it stops the task.

Any ideas on what to check if getting this in the logs?

I believe it has something to do with our health check to /v1/terrareg/health

When it runs it checks this healthcheck:  wget -qO- --server-response --no-check-certificate http://localhost:5000/v1/terrareg/health 2>&1 | grep -E '^\s+HTTP\/[0-9]+(\.[0-9]+)?\s+([0-9]+)\s+' | awk {'print $2'}

code 400, message Bad HTTP/0.9 request type ('\x16\x03\x01\x02\x00\x01\x00\x01ü\x03\x03séùD@\x9e\x8fZ\\Ï\xadË\x00/¿JTOpî8\x81Ø\x0eËg\x08?\x8d\x7f«|')

**Comment by mark-jones-rga** [comment, marked as answer]
I think I realized our error. We were running the healtcheck on https because we had modified flask to allow this.  Since synching back to upstream we no longer have that setup in flask!

**Reply by MatthewJohn** [reply]
Thanks @mark-jones-rga 
Apologies, had been away for the festive season :D
